src/components/PhotoUpload/ReportPhotoToolbar.js

```diff
@@ -0,0 +1,211 @@
+// @flow
+
+import { t } from 'c-3po';
+import * as React from 'react';
+import styled from 'styled-components';
+
+import Toolbar from '../Toolbar';
+
+import colors from '../../lib/colors';
+
+import CloseLink from '../CloseLink';
+import CustomRadio from '../NodeToolbar/AccessibilityEditor/CustomRadio';
+import StyledRadioGroup from '../NodeToolbar/AccessibilityEditor/StyledRadioGroup';
+
+import type { PhotoModel } from '../NodeToolbar/Photos/PhotoModel';
+
+
+export type ReportOptions = 'wrong-place' | 'outdated' | 'offensive' | 'other';
+
+const ReportValues : ReportOptions[] = ['wrong-place', 'outdated', 'offensive', 'other'];
+
+function shortReportCaption(reportValue: ReportOptions): ?string {
+  switch (reportValue) {
+    // translator: title for a report option where the photo does not match the place
+    case 'wrong-place': 
+      return t`Wrong Place`;
+    // translator: title for a report option where the photo does not match the current state of the place
+    case 'outdated': 
+      return t`Outdated`;
+    // translator: title for a report option where the photo is nsfw
+    case 'offensive': 
+      return t`Offensive or Inappropriate`;
+    // translator: title for a report option where the photo has another problem
+    case 'other': 
```

Comment:
I'm not 100% sure but I think the comments must preceed the lines with the strings for c-3po to understand them…

Reply:
Will fix!